nucypher/cli/commands/stake.py

```diff
@@ -948,3 +949,54 @@ def set_min_rate(general_config, transacting_staker_options, config_file, force,
                           receipt=receipt,
                           chain_name=blockchain.client.chain_name,
                           transaction_type='set_min_rate')
+
+
+@stake.command('create-workers')
+@group_staker_options
+@option_config_file
+@click.option('--cloudprovider', help="currently aws", default='aws')
+@click.option('--aws-profile', help="The AWS account profile you'd like to use", default='default')
+@click.option('--remote-provider', help="the blockchain provider for the remote node")
+@click.option('--clef', help="use clef for remote signing", default=False)
+@group_general_config
+def create_workers(general_config, staker_options, config_file, cloudprovider, aws_profile, remote_provider, clef):
+    """Creates workers for all stakes owned by the user for the given network."""
+
+    emitter = setup_emitter(general_config)
+
+    STAKEHOLDER = staker_options.create_character(emitter, config_file)
+
+    stakers = STAKEHOLDER.get_stakers()
+    if not stakers:
+        emitter.echo("No staking accounts found.")
+
+    stakes_to_deploy = []
+    for staker in stakers:
+        for stake in staker.stakes:
+            if stake.is_active and stake.worker_address == NULL_ADDRESS:
+                stakes_to_deploy.append((stake.staker_address, stake))
+
+
+    deployer = Deployers.get_deployer(cloudprovider)(emitter, STAKEHOLDER, aws_profile)
+    deployer.create_nodes_for_stakers(dict(stakes_to_deploy))
```

Comment:
A bit confused on the control here - There needs to be a single node for each staker, not stake.

Reply:
Really?  So a single worker can be bonded to multiple stakes?

Reply:
Yep! A single Staker has multiple stakes, and delegates a single worker for all of them by bonding. In the end there are only two addresses associated with an activate stake: One for the staker and one for the worker.